site/styles.css

```
@@ -9,35 +10,64 @@
+   --signal: oklch(0.66 0.16 42);
+   --signal-2: oklch(0.55 0.17 38);
+   --signal-3: oklch(0.86 0.06 60);
+
+   /* Per-product accent defaults */
+   --accent-otlp: var(--signal);
+   --accent-tsdb: var(--signal);
+   --accent-octo: var(--signal-2);
+   --accent-bench: var(--ink-2);
+
+  /* ── Typography ───────────────────────────────────────────────────────── */
   --mono: "JetBrains Mono", ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.9.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)